specs/swagger.yaml

```diff
@@ -198,7 +197,95 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-
+  /psms:
+     get:
+      summary: Get a collection of peptide spectrum matches  
+      operationId: getPSMs
+      description: Get specific Peptide Spectrum Match (PSM) if is present in the database and has been identified by a previous experiment in the resource. 
+      tags:
+        - spectra
+      produces:
+        - "application/json"
+      parameters:
+        - name: pageSize
+          in: query
+          description: How many items to return at one time (default 100, max 100)
+          required: false
+          type: integer
+          default: 100
+        - name: pageNumber
+          in: query
+          description: Current page to be shown paged datasets (default page 1)
+          required: false
+          type: integer
+          default: 0
+        - name: resultType
+          in: query
+          description: Type of the object to be retrieve Compact or Full dataset
+          required: true
+          type: string
+          enum: [compact, full]
+          default: compact
+        - name: usi
+          in: query
+          type: string
+          description: Universal Spectrum Identifier (USI)
+        - name: accession
+          in: query 
+          type: string 
+          description: Dataset accession 
+        - name: msrun 
+          in: query
+          type: string 
+          description: MsRun containing the spetra 
+        - name: filename 
+          in: query
+          type: string 
+          description: FileName containing the spectra
+        - name: scan
+          in: query 
+          type: string 
+          description: Scan to be search
+        - name: passThreshold
+          in: query
+          type: boolean 
+          description: the PSM pass the thorsholds of the repository (e.g. FDR thresholds)
+        - name: peptideSequence
+          in: query
+          type: string 
+          description: peptideSequence allows to retrieve all the PSMs for an specific Peptide Sequence including modified and un-modified previous. 
+        - name: proteinAccession 
+          in: query 
+          type: string
+          description: Protein Acession for the identified peptide
+        - name: charge
+          in: query
+          type: integer
+          description: charge state for the PSM
+        - name: modification
+          in: query
+          type: string 
+          description: modification that found in the peptide. For example, to query all peptides that are oxidated. 
```

Comment:
Remove "that". Also, you may want to pluralize "modifications", since if I recall correctly the API will allow multiple modifications to be specified in the value of this parameter.